README.md

```diff
@@ -238,6 +238,38 @@ If no AES private key file aes_sec is found in the intended location or if the f
 no encryption of the sensitive variables will take place. All data will stored in plain text within
 the JVM.
 
+### Version Configuration For Terraform Providers
+
+Xpanse currently uses Terraform to orchestrate resources on cloud providers, but versions of
+Terraform for cloud providers are in constant iteration. In order to configure the Terraform version
+of the cloud providers more conveniently and quickly, you can configure the Terraform version of the
+cloud providers in the xpanse configuration file `resource/application.properties`
```

Comment:
please give exact link to this file in the repo. (you can link repo files easily in the readme doc)